components/brave_rewards/browser/rewards_service_impl_unittest.cc

```diff
@@ -87,6 +111,54 @@ class RewardsServiceTest : public testing::Test {
   base::ScopedTempDir temp_dir_;
 };
 
-// add test for strange entries
+TEST_F(RewardsServiceTest, GetExternalWallet) {
```

Comment:
these tests need to be behind the buildflag

Reply:
resolved